src/Sql/Sql/Database Backup/Cmdlet/UpdateAzureSqlDatabaseLongTermRetentionBackup.cs

```diff
@@ -95,6 +101,11 @@ public class UpdateAzureSqlDatabaseLongTermRetentionBackup : AzureSqlDatabaseLon
             ValueFromPipelineByPropertyName = true,
             Position = 3,
             HelpMessage = "The name of the backup.")]
+        [Parameter(Mandatory = true,
+            ParameterSetName = UpdateBackupStorageAccessTierSet,
+            ValueFromPipelineByPropertyName = true,
+            Position = 3,
```

Comment:
Is the `Position = 3` necessary here? We recommand to have less than 3 parameters with position, this is why the analysis gives an error.

Reply:
ACK I'll change it to Named